spec/Cabin/Seat.vspec

```diff
@@ -182,147 +213,300 @@ Airbag.IsDeployed:
   datatype: boolean
   type: actuator
   description: Is switch for less lumbar support engaged (SingleSeat.Backrest.Lumbar.Support).
+  deprecation: v5.0 - refactored to Seat.Backrest.IsLessLumbarSupportSwitchEngaged
+
+Backrest.IsLessLumbarSupportSwitchEngaged:
+  datatype: boolean
+  type: actuator
+  description: Is switch for less lumbar support engaged.
 
 Switch.Backrest.Lumbar.IsUpEngaged:
   datatype: boolean
   type: actuator
   description: Lumbar up switch engaged (SingleSeat.Backrest.Lumbar.Support).
+  deprecation: v5.0 - refactored to Seat.Backrest.IsLumbarUpSwitchEngaged
+
+Backrest.IsLumbarUpSwitchEngaged:
+  datatype: boolean
+  type: actuator
+  description: Lumbar up switch engaged.
 
 Switch.Backrest.Lumbar.IsDownEngaged:
   datatype: boolean
   type: actuator
   description: Lumbar down switch engaged (SingleSeat.Backrest.Lumbar.Support).
+  deprecation: v5.0 - refactored to Seat.Backrest.IsLumbarDownSwitchEngaged
+
+Backrest.IsLumbarDownSwitchEngaged:
+  datatype: boolean
+  type: actuator
+  description: Lumbar down switch engaged.
 
 Switch.Backrest.SideBolster:
   type: branch
   description: Switches for SingleSeat.Backrest.SideBolster.
+  deprecation: v5.0 - nested properties refactored to Seat.Backrest.<Property>
 
 Switch.Backrest.SideBolster.IsMoreSupportEngaged:
   datatype: boolean
   type: actuator
   description: Is switch for more side bolster support engaged (SingleSeat.Backrest.SideBolster.Support).
+  deprecation: v5.0 - refactored to Seat.Backrest.IsMoreSideBolsterSupportSwitchEngaged
+
+Backrest.IsMoreSideBolsterSupportSwitchEngaged:
+  datatype: boolean
+  type: actuator
+  description: Is switch for more side bolster support engaged.
 
 Switch.Backrest.SideBolster.IsLessSupportEngaged:
   datatype: boolean
   type: actuator
   description: Is switch for less side bolster support engaged (SingleSeat.Backrest.SideBolster.Support).
+  deprecation: v5.0 - refactored to Seat.Backrest.IsLessSideBolsterSupportSwitchEngaged
+
+Backrest.IsLessSideBolsterSupportSwitchEngaged:
+  datatype: boolean
+  type: actuator
+  description: Is switch for less side bolster support engaged.
 
 Switch.Seating:
   type: branch
   description: Describes switches related to the seating of the seat.
+  deprecation: v5.0 - nested properties moved to Seat.Seating.<Property>.
 
 Switch.Seating.IsForwardEngaged:
   datatype: boolean
   type: actuator
   description: Is switch to increase seating length engaged (SingleSeat.Seating.Length).
+  deprecation: v5.0 - refactored to Seat.Seating.IsForwardSwitchEngaged
+
+Seating.IsForwardSwitchEngaged:
+  datatype: boolean
+  type: actuator
+  description: Is switch to increase seating length engaged.
 
 Switch.Seating.IsBackwardEngaged:
   datatype: boolean
   type: actuator
   description: Is switch to decrease seating length engaged (SingleSeat.Seating.Length).
+  deprecation: v5.0 - refactored to Seat.Seating.IsBackwardSwitchEngaged
+
+Switch.IsBackwardSwitchEngaged:
```

Comment:
Shall be `Seating.IsBackwardSwitchEngaged`?

Reply:
You are right. I missed that one. I will push the change.